metadata/de-DE/title.txt

```diff
@@ -0,0 +1 @@
+Jamulus (Zuhörer Client)
```

Comment:
I don't really like this translation

Reply:
Don't have a good suggestion either.
- Wiedergabe-Client
- Player
- Passiver Client

Reply:
Deos it need an adjective?  If we're not supporting installation as server on Android, I guess that does need stating, though.

Reply:
(I mean "Jamulus (Client)" would be good enough, right?  Looks like "Kunde" wouldn't be the right term.)

Reply:
```suggestion
Jamulus (Wiedergabe-Client)
```

I think that fits best.